__tests__/app.test.js

```diff
@@ -67,6 +67,135 @@ describe("/api/articles", () => {
         });
       });
   });
+
+  test("GET?sort_by=: 200 -- responds with an array sorted by the given column", () => {
```

Comment:
I see what you were going for here but this is perhaps a little overkill, fewer tests would probably suffice to ensure that the functionality is working as expected